src/utils/PageParsing/MavenApache.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//https://repo.maven.apache.org/maven2/commons-collections/commons-collections/3.2.1/
+//"purl": "pkg:maven/com.mycompany.myproduct/artifact-name@2.1.7",
+import {PackageURL} from 'packageurl-js';
+import {FORMATS} from '../Constants';
+import {
+  generatePackageURL,
+  generatePackageURLWithNamespace,
+  generatePackageURLComplete
+} from './PurlUtils';
+
+//pkg:type/namespace/name@version?qualifiers#subpath
+//Sonatype expects: "packageUrl": "pkg:maven/org.yaml/snakeyaml@1.17?type=jar"
+const parseMavenApache = (url: string): PackageURL | undefined => {
+  // console.trace("parseMavenApache", url)
```

Comment:
No need to comment this out if you have the log level at trace!

Reply:
OK, will restore them